queries.lisp

```diff
@@ -1,10 +1,85 @@
 (in-package #:org.shirakumo.tooter)
 
+(defun pagination-url (handle)
+  (cdr handle))
+
+(defun pagination-decoding-function (handle)
+  (car handle))
+
+(defun make-pagination-handle (decoding-function url)
+  (cons decoding-function url))
+
+(defmacro with-return-also-headers ((decoding-function) &body body)
```

Comment:
A better name would be `with-pagination-return`

Reply:
Sure, i agree it is a much better name, fixed!